pennylane/estimator/resource_operator.py

```diff
@@ -270,16 +270,31 @@ def resource_rep_from_op(self) -> CompressedResourceOp:
     def resource_decomp(cls, *args, **kwargs) -> list[GateCount]:
         r"""Returns a list of actions that define the resources of the operator."""
 
+    # pylint: disable=import-outside-toplevel
     @classmethod
     def adjoint_resource_decomp(cls, target_resource_params: dict | None = None) -> list[GateCount]:
         r"""Returns a list representing the resources for the adjoint of the operator.
 
+        For ResourceOperators that don't define a `adjoint_resource_decomp`, this will be the default
+        `adjoint_resource_decomp` method. Every operator in the decomposition will be adjointed.
```

Comment:
Same suggestion as below to add a dedicated `Resources` section where we explain how the default adjoint resources are computed.

```suggestion
        For ``ResourceOperators`` that don't define a ``adjoint_resource_decomp``, this will be the default
        ``adjoint_resource_decomp`` method.
```